torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4353,6 +4377,100 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.mode,
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            sample_inputs_func=sample_inputs_mode,),
+    UnaryUfuncInfo('mvlgamma',
+                   ref=reference_mvlgamma if TEST_SCIPY else _NOTHING,
+                   variant_test_name='mvlgamma_p_1',
```

Comment:
One other option to improve readability here would be to make an mvlGammaInfo class that inherits from UnaryUfuncInfo. It could populate most of this info by default. That would slim down each entry and make it easier to add different `p` values